core/domain/collection_domain.py

```diff
@@ -375,6 +483,16 @@ def _migrate_to_latest_yaml_version(cls, yaml_content):
 
     @classmethod
     def from_yaml(cls, collection_id, yaml_content):
+        """Converts a YAML to Collection domain object.
```

Comment:
YAML --> YAML string (otherwise it's like you're saying, of a code file, "Converts a Python to a ...")